client/src/components/FullScreenMessage.js

```diff
@@ -0,0 +1,17 @@
+import { PropTypes } from "prop-types";
+import React from "react";
+
+const FullScreenMessage = ({ id, children }) => {
+  return (
+    <section className="fullscreen-message" id={id}>
+      <div>{children}</div>
+    </section>
+  );
+};
```

Comment:
As this only a "rendering" component, it like to do like this (but do as you want :smile:)
```suggestion
const FullScreenMessage = ({ id, children }) => (
  <section className="fullscreen-message" id={id}>
    <div>{children}</div>
  </section>
);
```